core/urls.py

```
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Ruff (0.12.2)

UTF-8 encoding declaration is unnecessary

Remove unnecessary coding comment

(UP009)